protocol/packages/protocol/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+use std::collections::BTreeSet;
+
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+#[serde(deny_unknown_fields, rename_all = "snake_case")]
+pub struct Protocol {
+    pub dex_network: String,
+    pub dex: String,
+    pub liquidity_provider_currency_ticker: String,
```

Comment:
LPN is a known abbreviation for the project and in its lowercase form is used throughout the source code, please rename